src/macropad/index.html

```diff
@@ -0,0 +1,175 @@
+    <meta name="description" content="Resources - PDXKBC" />
+    <meta name="og:description" content="Resources - PDXKBC" />
+    <meta name="og:title" content="Resources - PDXKBC" />
+    <title>Resources - PDXKBC</title>
+    <meta name="twitter:card" content="summary" />
+    <meta name="twitter:site" content="@wavebeem" />
+    <meta name="twitter:creator" content="@wavebeem" />
+    <meta name="og:image" content="https://www.pdxkbc.com/logo.jpg" />
+  </head>
+  <body>
+    <header>
+      <nav class="content">
+        <img src="/logo.svg" alt="PDXKBC" class="logo" width="28" height="28" />
+        <a href="/">Home</a>
+        <a href="/gallery">Gallery</a>
+        <a href="/macropad">Macropad</a>
+        <a href="/resources">Resources</a>
+        <a href="/coc">Conduct</a>
+        <!-- TODO:  Add link for resources page to header of all pages   -->
+      </nav>
+    </header>
+
+    <main>
+      <h1>PDXKBC Macropad</h1>
+      <ul>
+        <li><a href="#macropad">Overview</a></li>
+	<li><a href="#links">Links</a></li>
+        <li><a href="#parts_list">Parts List</a></li>
+        <li><a href="#build_guide">Build Guide</a></li>
+      </ul>
+
+      <h2 id="macropad">Overview</h2>
+      <figure>
+        <img src="macropad.jpg" width="600" height="657" />
+        <figcaption>PDXKBC macropad designed and built by Franklin</figcaption>
+      </figure>
+
+      <h2 id="links">Links</h2>
+      <ul>
+        <li>
+          <a href="https://github.com/fharding1/pdxkbc-badge"
+            >Hardware GitHub repository</a
+          >
+        </li>
+        <li>
+          <a
+            href="https://github.com/qmk/qmk_firmware/tree/master/keyboards/pdxkbc"
+            >QMK GitHub repository</a
+          >
+        </li>
+      </ul>
+
+      <h2 id="parts_list">Parts List</h2>
+      <figure>
+        <img src="materials.jpg" width="600"/>
+      </figure>
+      <ul>
+        <li>
+          Badge and Case PCBs (See <a href="#links">Links</a> section for source files)
+        </li>
+        <li>
+          Standoffs:
+        </li>
+          <ul>
+            <li>4x M4 10+6mm brass Male-Female Spacers</li>
+            <li>4x M3 x 6mm screws</li>
+            <li>4x M3 nuts</li>
+          </ul>
+        <li>
+          1x 5v Arduino Pro Micro
+        </li>
+        <li>
+          6x 1N4148 Diodes
+        </li>
+        <li>
+          4x Rubber Bumpers/Feet
+        </li>
+        <li>
+          6x PCB Mount Switches
+        </li>
+        <li>
+          6x 1u Keycaps
+        </li>
+      </ul>
+
+      <h2 id="build_guide">Build Guide</h2>
+
+      <h3>Tools</h3>
+      <p>You'll need</p>
+      <ul>
+        <li>A soldering iron and solder</li>
+        <li>A phillips screwdriver</li>
+        <li>Flush cutters</li>
+      </ul>
+
+      <h3>Assembly</h3>
+      <h4>Diodes</h4>
+      <p>Bend your diodes as shown</p>
+      <figure>
+        <img src="bentdiode.jpg" width="600"/>
```

Comment:
Images should have an `alt` value for accessibility, but u can omit it with `alt=""` if you supply a figcaption